precompile/allow_list.go

```diff
@@ -43,12 +40,17 @@ var (
 
 // AllowListConfig specifies the initial set of allow list admins.
 type AllowListConfig struct {
-	AllowListAdmins []common.Address `json:"adminAddresses"`
+	AllowListAdmins  []common.Address `json:"adminAddresses"`
+	EnabledAddresses []common.Address `json:"enabledAddresses,omitempty"` // initial enabled addresses
```

Comment:
```suggestion
	EnabledAddresses []common.Address `json:"enabledAddresses"` // initial enabled addresses
```

Adding `omitempty` here will only result in setting `EnabledAddresses` to `nil` instead of an empty list. Since we don't differentiate between these two within the allow list, we should keep it consistent with `AllowListAdmins`.